src/wallet/feebumper.cpp

```diff
@@ -281,21 +282,35 @@ Result CreateRateBumpTransaction(CWallet& wallet, const uint256& txid, const CCo
     // We cannot source new unconfirmed inputs(bip125 rule 2)
     new_coin_control.m_min_depth = 1;
 
-    constexpr int RANDOM_CHANGE_POSITION = -1;
-    auto res = CreateTransaction(wallet, recipients, RANDOM_CHANGE_POSITION, new_coin_control, false);
-    if (!res) {
-        errors.push_back(Untranslated("Unable to create transaction.") + Untranslated(" ") + util::ErrorString(res));
-        return Result::WALLET_ERROR;
-    }
+    if (recipients.size() > 0) {
+        constexpr int RANDOM_CHANGE_POSITION = -1;
+        auto res = CreateTransaction(wallet, recipients, RANDOM_CHANGE_POSITION, new_coin_control, false);
+        if (!res) {
+            errors.push_back(Untranslated("Unable to create transaction.") + Untranslated(" ") + util::ErrorString(res));
+            return Result::WALLET_ERROR;
+        }
 
-    const auto& txr = *res;
-    // Write back new fee if successful
-    new_fee = txr.fee;
+        const auto& txr = *res;
+        // Write back new fee if successful
+        new_fee = txr.fee;
 
-    // Write back transaction
-    mtx = CMutableTransaction(*txr.tx);
+        // Write back transaction
+        mtx = CMutableTransaction(*txr.tx);
 
-    return Result::OK;
+        return Result::OK;
```

Comment:
0b280affd4c755d8f9eb809b3676389b2c2c52c0 nit: `return` could be outside the `if else`. If you touch it, there's also a typo in the commit description: `transcation`